src/olympia/bandwagon/views.py

```diff
@@ -677,8 +677,11 @@ def get_object(self):
         return super(CollectionAddonViewSet, self).get_object()
 
     def get_queryset(self):
-        qs = CollectionAddon.objects.filter(
-            collection=self.get_collection_viewset().get_object())
+        qs = (
+            CollectionAddon.objects
+            .filter(collection=self.get_collection_viewset().get_object())
+            .prefetch_related('addon'))
```

Comment:
Do we want a test somewhere that counts the queries to make sure we don't regress this later?  Or will that be added when we get the counts down further.

Reply:
My hope was that I add that later once I can make the counts much more predictable :goat:

Reply:
I'll get that done asap though and promise tests!